app/src/features/Columns/ColumnSelect.tsx

```diff
@@ -61,14 +65,28 @@ const ColumnSelects = ({
       skip: !credentials?.[0],
     }
   );
+  const { data: mapping } = useApiResourcesRetrieveQuery(
```

Comment:
Isn't the mapping in the app state? This would avoid an API call and it's how it's done in `FilterJoinList.tsx`, I think

Reply:
I can't find it but i'll add it